marketplace/netlify-build/src/index.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+import { init } from 'contentful-ui-extensions-sdk';
+import { createPubSub } from './pubnub-client';
+import { Select, Option } from '@contentful/forma-36-react-components';
+import NeflifySideBarBuildButton from './NetlifySideBarBuildButton';
+
+function createSiteStructure(parameters = {}) {
+  const buildHooks = (parameters.buildHookUrls || '').split(',');
+  const channels = (parameters.channels || '').split(',');
```

Comment:
Actually channel names need to:

1. be stable
2. be globally unique per installation per site

Now I see we actually don't need it as a parameter. I would use suffixes of build hook URLs:

```js
const buildHookUrls = readCsvParam(paramters.buildHookUrls)
const channels = buildHookUrls.map(hook => hook.split('/').pop())
```

Reply:
I want to make sure I'm following 100% here. Are you saying that we can stop creating the channel names in the main UI code and just have this UIE create the channel names here?

Reply:
Followed up offline, this is the case.